lesson04/assignment/css/_base.scss

```diff
@@ -0,0 +1,17 @@
+body, html, ul, li, img, div, nav {
+	margin: 0;
+	padding: 0;
+}
+html {
+    $bk-color : #95DEE3;  /* For browsers that do not support gradients */
+    /* background-image: linear-gradient(red, yellow); */
+    background-image: linear-gradient(-10deg, #C62828, #BA68C8);
+  /*  min-height: 100vh; */
+}
+body {
+	width: 80%;
```

Comment:
Rather than set the body to 80%, you would want to use a container and give it that max-width value and left/right margin of auto to center the content.